### DISCUSSION TITLE
别看今天闹得欢

### DISCUSSION THREAD

**Discussion by wd30x** [WeNeedHome/SummaryOfLoanSuspension#944] [General]
实际上啊那些错误执行者，它也是有一本账的，这个帐是记在那儿的。一旦他出事了，这个帐全给你拉出来了。别看你今天闹得欢，小心啊今后拉清单，这都得应验的，不要干这种事情，头上三尺有神明，一定要有敬畏之心！

**Comment by Liboce** [comment]
经典，正确的，为新时代维稳工作指明方向

**Comment by tanjunwen0302** [comment]
哪有什么救世主？哪有什么神明？
